src/slideseq/plot/plot_downsampling.py

```diff
@@ -131,11 +133,25 @@ def model_least_squares(params, *, r, data):
     ax.set_ylabel("Transcripts per matched barcode")
     ax.set_title("Average transcripts for matched barcodes")
 
-    # r went upto 1.0 for actual data, but x_values for model go up to 3.0
-    ax.set_xlim(0.0, 3.1)
+    ax.set_xlim(
+        0.0, 3.1
+    )  # r went upto 1.0 for actual data, but x_values for model go up to 3.0
```

Comment:
The inline comment made `black` format this weirdly, so it's nicer like this (also, a typo)
```suggestion
    # r went up to 1.0 for actual data, but x_values for model go up to 3.0
    ax.set_xlim(0.0, 3.1)
```